flipperdevices/flipperzero-firmware
# [FL-3401,  FL-3402] SubGhz:  add "SubGhz test" external application and the ability to work "SubGhz" as an external application

# What's new

- [FL-3401] SubGhz: add "SubGhz test" external application
- SubGhz: delete test functionality from SubGhz app (+5Kb to a free flash)
- [FL-3402] SubGhz: move func protocol creation API

# Verification 

- Check application Aplications -> Debug -> Sub-Ghz Test
- Build Sub-Ghz as an external application and check if it works

# Checklist (For Reviewer)



[FL-3401]: https://flipperzero.atlassian.net/browse/FL-3401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FL-3402]: https://flipperzero.atlassian.net/browse/FL-3402?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
